draft-ietf-teep-architecture.md

```diff
@@ -1329,24 +1329,26 @@ may need to be updated.
 ## Compromised TAM
 
 Device TEEs are responsible for validating the supplied TAM certificates
-to determine that the TAM is trustworthy. A compromised TAM can install
-arbitrary TAs to, potentially, many devices, which makes TAMs high value
-targets. An advanced, persistent threat actor could potentially compromise
-a TAM without impacting its certificate or raising concern from the TAM's
-operator. To mitigate this threat, TEEs have several options for 
-detecting and mitigating a compromised TAM:
-
-1. Apply an ACL to the TAM, indicating which TAs it is permitted to install.
-2. Use a transparency log to expose a TA compromise: TAMs publish an 
-out-of-band record of TA releases, allowing a TEE to cross-check the TAs
-delivered against the TA releases in order to detect a TAM compromise.
+to determine that the TAM is trustworthy. A compromised TAM may bring multiple threats 
```

Comment:
```suggestion
. A compromised TAM may bring multiple threats 
```

Reply:
I removed the second part of the sentence since the subsequent text explains why just verifying the TAM certificate is not enough for establishing trustworthiness